hw/ip/prim/dv/prim_present/prim_present_sim_cfg.hjson

```diff
@@ -45,6 +45,7 @@
     {
       name: smoke
       tests: ["prim_present_smoke"]
+      run_opts: ["+smoke_test=1"]
```

Comment:
nit: Now that `smoke_test` is a "mode" in `prim_present_smoke` test, it no longer makes sense to call the test `prim_present_smoke`. Perhaps rename the test to `prim_present_test`. [Update testplan and other areas as applicable].

Reply:
I just realized that PRESENT still has a testplan for some reason...let me also remove that from this PR.